clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/model/AzureServerGroupDescription.groovy

```diff
@@ -73,6 +88,37 @@ class AzureServerGroupDescription extends AzureResourceOpsDescription {
     String.format("%s-%s-%s", application, stack, detail)
   }
 
+  String getLoadBalancerName() {
+    loadBalancerName ?: "none"
```

Comment:
This is not going to fly; setting the load balancer name to anything but null will break the destroy/delete operation; I would rather prefer setting it to the name of the server group per the create operation.